source/Body.h

```diff
@@ -78,7 +78,9 @@ class Body {
 	// Set the color swizzle.
 	void SetSwizzle(int swizzle);
 
-	double Alpha() const;
+	double Alpha(const Point &drawCenter) const;
+	double DistanceAlpha(const Point &drawCenter) const;
+	bool IsVisible(const Point &drawCenter) const;
```

Comment:
plz add at least one comment for the bunch

we have a comment for // Set the color swizzle. and this isnt more explicit than that lol

Reply:
comment added!